content/blog/copilot-lessons/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Engineering Lessons from Building Pulumi Copilot"
+date: 2024-12-12T13:56:13-05:00
+draft: false
+meta_desc: Learn key engineering lessons from building Pulumi Copilot, including how to minimize LLM workload, validate outputs, and deal with hallucination.
+meta_image: meta.png
+authors:
+    - artur-laksberg
+    - simon-howe
```

Comment:
@foot IF you are ok being listed as an author we will need a picture to use for you here.